back/sssscs/src/main/java/com/ib/user/User.java

```diff
@@ -61,6 +60,11 @@ public enum Role {
 	private Boolean blocked = false;
 	@Column(nullable = true)
 	private LocalDateTime blockEndDate;
+	@Column(nullable = true)
+	private LocalDateTime lastTimeOfPasswordChange = null;
```

Comment:
When should this be `null`? Even when registering the account, we still set this field in `setNewPassword`

Reply:
It's non-null so that I don't have to initialize it in `data.sql`.